src/kubernetes/kafka/kafka-deployment.yaml

```diff
@@ -3,35 +3,33 @@ kind: Deployment
 metadata:
   name: kafka-broker
   labels:
-    app: kafka
+    app: kafka-broker
     component: kafka-broker
 spec:
   replicas: 1
   template:
     metadata:
       labels:
-        app: kafka
-        component: kafka-broker
+        app: kafka-broker
+        component: kafka-broker-c
     spec:
       containers:
-      - name: kafka
-        image: wurstmeister/kafka:2.11-0.11.0.3
+      - name: kafka-broker
+        image: confluentinc/cp-kafka:5.3.1
         ports:
         - containerPort: 9092
         env:
+          - name: KAFKA_ADVERTISED_LISTENERS
+            value: PLAINTEXT://kafka-broker:9092
           - name: ENABLE_AUTO_EXTEND
             value: "true"
           - name: KAFKA_RESERVED_BROKER_MAX_ID
             value: "999999999"
-          - name: KAFKA_AUTO_CREATE_TOPICS_ENABLE
-            value: "false"
-          - name: KAFKA_PORT
-            value: "9092"
-          - name: KAFKA_ADVERTISED_PORT
-            value: "9092"
           - name: KAFKA_ADVERTISED_HOST_NAME
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
           - name: KAFKA_ZOOKEEPER_CONNECT
             value: kafka-zk:2181
+          - name: KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR
```

Comment:
I guess we set this as it is required for the single node cluster?

Reply:
Removed this property.